app/controllers/concerns/api_redirectable.rb

```diff
@@ -0,0 +1,15 @@
+module ApiRedirectable
+  extend ActiveSupport::Concern
+
+  included do
+    before_filter :redirect_if_api_request
+  end
+
+  def redirect_if_api_request
+    redirect_to "/api/#{slug_param}.json" if request.format.json?
```

Comment:
for the avoidance of uncertainty, is it also worth guarding this with a test for `request.get?`?